packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/util.test.ts

```diff
@@ -100,3 +109,136 @@ describe("getDefaultFigureForType", () => {
         });
     });
 });
+
+describe("generateLockedFigureAppearanceDescription", () => {
+    // one argument
+    test.each([
+        ["grayH"],
+        ["red"],
+        ["blue"],
+        ["green"],
+        ["purple"],
+        ["orange"],
+        ["pink"],
+    ])(
+        `should return a string with color of %s and a solid stroke style`,
+        (color: LockedFigureColor) => {
+            const description =
+                generateLockedFigureAppearanceDescription(color);
+            const convertedColor = color === "grayH" ? "gray" : color;
+            expect(description).toBe(`. Appearance solid ${convertedColor}.`);
```

Comment:
Thanks for adding these thorough tests!

I think using a ternary in the test might be a bit overkill, though — it makes the tests harder to read, and in a way tautological, since now they duplicate the same logic that's in the production code.

An alternative would be to have just two tests: one that exemplifies the "normal" logic of passing the color name through unchanged, and one that calls out the special case of `grayH`:

```ts
it("describes a solid color", () => {
  const description =
    generateLockedFigureAppearanceDescription("red");
  expect(description).toBe(". Appearance solid red");
});

it("describes grayH as gray", () => {
  const description =
    generateLockedFigureAppearanceDescription("grayH");
  expect(description).toBe(". Appearance solid gray");
});
```